### PR TITLE
frontend_design

### PR DESCRIPTION
**Pull request by hsupeiyu12**
Add a page to submit the review
Add feedback item structure
Add comparison page
Revise the navigation bar

### CODE REVIEW

File: Education_Pathways/frontend/public/_formhelpers.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% macro render_field(field) %}
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/public/_formhelpers.html:1]: Which part is the "field"? Could you add a short comment?

File: Education_Pathways/frontend/src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+            </option>
+            <option value="University of Toronto Scarborough">
+              University of Toronto Scarborough
+            </option>
+          </select>
+        </div>
+        <div style={{ marginTop: "1%" }}>
+          <label> Department: </label>
+          <select
+            value={this.state.department}
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/Form.js:107]: I think you don't need to add the selections manually. I'll change this part in my pull request.

File: Education_Pathways/frontend/src/components/Navbar.js [hunk 3]
```diff
@@ -21,27 +52,36 @@ function useQuery() {
...
+  // componentDidMount() {
+  //   if (localStorage.getItem('username') !== "") {
+  //     this.setState({username: localStorage.getItem('username')})
+  //   }
+  // }
+
+  // logOut = () => {
+  //   localStorage.setItem('username', "");
+  //   this.setState({username: ""})
+  // }
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/Navbar.js:84]: delete the commented out lines

File: Education_Pathways/frontend/src/components/Navbar.js [hunk 4]
```diff
@@ -58,14 +98,23 @@ export default class NavbarComp extends Component {
...
-                <Nav.Link as={Link} to="/about">
+                <Nav.Link className={classes.navpath} as={Link} to="/about">
                   About Us
                 </Nav.Link>
 
-                {/* <Nav.Link href="/search" style={{ color: "white", display: "inline" }}>
-                  Search
+                {/* <Nav.Link as={Link} to="/filter">
+                  Filter
                 </Nav.Link> */}
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/Navbar.js:107]: delete the commented out page

File: Education_Pathways/frontend/src/components/Navbar.js [hunk 5]
```diff
@@ -94,17 +143,45 @@ We are looking for feedback to improve Education Pathways and make it more usefu
...
             <Route path="/">
               <SearchResultDisplay />
+              <section>
+                <FeedbackList feedbacks={DUMMY_DATA}/>
+              </section>
             </Route>
 
+            
+            
+
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/Navbar.js:184]: delete unused lines

File: Education_Pathways/frontend/src/components/ResultDisplay.jsx [hunk 6]
```diff
@@ -33,21 +35,33 @@ class SearchResultDisplay extends Component{
...
             result_temp.push(<Label></Label>)
             for (let i = 0; i < len; i++) {
                 result_temp.push(<Result course_code={res.data[i].code} course_name={res.data[i].name}></Result>)
-            }
+                let dict={};
+                dict['coursename'] = res.data[i].code;
+                dict['workload'] = res.data[i].name;
+                global_array.push(dict);
+   
+                //result_temp.push(<button> + </button>)
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/ResultDisplay.jsx:51]: clean code

File: Education_Pathways/frontend/src/components/ResultDisplay.jsx [hunk 7]
```diff
@@ -33,21 +35,33 @@ class SearchResultDisplay extends Component{
...
+              }
             this.setState({results: result_temp})
           } else if (res.data.length === 0) {
             alert("Course not found")
           }else {
             let result_temp = []
             result_temp.push(<Label></Label>)
             result_temp.push(<Result course_code={res.data.course.code} course_name={res.data.course.name}></Result>)
+            //result_temp.push(<button> + </button>)
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/ResultDisplay.jsx:60]: clean code, delete the comment

File: Education_Pathways/frontend/src/components/css/navbar.css [hunk 8]
```diff
@@ -3,16 +3,26 @@
...
+.nav-item a:link { text-decoration: none; }
+
+.nav-item a:visited { text-decoration: none; }
+
+
+.nav-item a:hover { text-decoration: none; color: #c4d1e4;}
+
+
+.nav-item a:active { text-decoration: none; } 
+ */
```
Comment (EmmaZhao0413) [inline Education_Pathways/frontend/src/components/css/navbar.css:23]: delete the commented out lines

**Review by Eric-R-Liang — COMMENTED** [review]
Reviewed. Need to add comments to major changes such as Form.js and Navbar.js. Also, all the current changes in this branch for the Comparison page will be lost after merging with comparison branch. Need to discuss what changes you want to keep, if any.

**Review by EmmaZhao0413 — COMMENTED** [review]
codes are well formatted and easy to understand. There are comments not cleaned. Please clean the code before merge the PR.

File: Education_Pathways/frontend/public/_formhelpers.html (hunk 1 above)
Comment (hsupeiyu12) [reply Education_Pathways/frontend/public/_formhelpers.html:1]: will add on the next pull request

**Comment by hsupeiyu12** [comment]
will start a new pull request to clean the code

**Comment by hsupeiyu12** [comment]
> Reviewed. Need to add comments to major changes such as Form.js and Navbar.js. Also, all the current changes in this branch for the Comparison page will be lost after merging with comparison branch. Need to discuss what changes you want to keep, if any.

Understood. Form.js will be deleted.

**Pull request merged**